src/components/BreakoutRoomsEditor/SelectableParticipant.vue

```diff
@@ -4,35 +4,47 @@
 -->
 
 <template>
-	<div tabindex="0"
-		class="selectable-participant">
-		<input id="participant.attendeeId"
+	<label :for="participant.attendeeId"
+		class="selectable-participant"
+		:class="{ 'selectable-participant--selected': selected }">
+		<input :id="participant.attendeeId"
 			v-model="modelProxy"
 			:value="participant.attendeeId"
 			type="checkbox"
-			name="participant.attendeeId">
+			class="hidden-visually">
 		<!-- Participant's avatar -->
 		<AvatarWrapper :id="participant.actorId"
 			:name="participant.displayName"
 			:source="participant.source || participant.actorType"
 			disable-menu
 			disable-tooltip
-			show-user-status
-			show-user-status-compact />
-		<div>
-			{{ participant.displayName }}
-		</div>
-	</div>
+			show-user-status />
+
+		<span class="selectable-participant__content">
+			<span class="selectable-participant__content-name">
+				{{ participant.displayName }}
+			</span>
+			<span v-if="participant.statusMessage"
+				class="selectable-participant__content-subname">
+				{{ participant.statusIcon + ' ' + participant.statusMessage }}
+			</span>
+		</span>
+
+		<IconCheck v-if="selected" class="selectable-participant__check-icon" :size="20" />
```

Comment:
It's sad, we cannot use Cannot we use `NcListItem` here? Because of the check mark? Even with extra actions slot?

Reply:
I thought the problem was in the `<a>` tag? Otherwise we can try to hide input in extra slot, I think

Reply:
Postponing until NcListItem is a11y-aproppriate to do that. Then we could migrate this component as well

Reply:
Yet, we are using NcListItem in selectable participants in new conversation dialog. Can we use the same ?

Reply:
Want to do it with separate pr, to split the changes